ZenPacks/zenoss/OpenStackInfrastructure/datasources/HeartbeatsAMQPDataSource.py

```diff
@@ -78,6 +70,7 @@ class HeartbeatsAMQPDataSourceInfo(AMQPDataSourceInfo):
 
 
 class HeartbeatsAMQPDataSourcePlugin(AMQPDataSourcePlugin):
+    """This class is deprecated. Do not use!"""
     proxy_attributes = ('expected_ceilometer_heartbeats',)
     exchange_name = '$OpenStackInboundHeartbeats'
     queue_name = "$OpenStackInboundHeartbeat"
```

Comment:
These values would cause problems (if used)- these have been removed from the .qjs and so are no longer valid values.. might as well remove them..  they aren't used anyway since you don't call the parent class's collect method anyway.